cmd/zpool/zpool_main.c

```diff
@@ -2861,10 +2896,20 @@ print_iostat_header_impl(iostat_cbdata_t *cb, unsigned int force_column_width,
 
 	print_iostat_labels(cb, force_column_width, iostat_top_labels);
 
+	if (cb->cb_kind)
+		printf("      ");
+
+	printf("\n");
+
 	printf("%-*s", namewidth, title);
 
 	print_iostat_labels(cb, force_column_width, iostat_bottom_labels);
 
+	if (cb->cb_kind)
+		printf("  kind");
```

Comment:
That would seem to complicate things, sending some extra arguments, and ending up outside the loop in `print_iostat_labels()`.  Perhaps if `cb_kind` is turned into a `cb_flags`...  Should I look into that?